BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -0,0 +1,544 @@
+version 1.0
+import "BenchmarkVCFs.wdl" as Benchmark
+
+workflow FindSamplesAndBenchmark {
+    
+    input {
+        Array[File] input_callset
+        Array[File] ground_truth_files
+        Array[File] ground_truth_indexes
+        Array[File] ground_truth_intervals
+        Array[String] truth_labels
+        Array[File] annotation_intervals
+
+        File? gatkJarForAnnotation
+        String? annotationName
+  
+        File ref_fasta
+        File ref_fasta_index
+        File ref_fasta_dict
+        File ref_fasta_sdf
+
+        File haplotype_database 
+        File picard_cloud_jar
+
+        String docker 
+
+        File? interval_list_override = "gs://concordance/interval_lists/chr9.hg38.interval_list"
+        File? runs_file_override = "gs://concordance/hg38/runs.conservative.bed"
+        String comparison_docker
+
+        String? analysis_region
+
+        Boolean remove_symbolic_alleles=false
+
+        Array[File] stratIntervals = [
+                         "gs://concordance/hg38/runs.conservative.bed",
+                         "gs://concordance/hg38/LCR-hs38.bed",
+                         "gs://concordance/hg38/mappability.0.bed",
+                         "gs://concordance/hg38/exome.twist.bed"]
+        Array[String] stratLabels = [
+                         "HMER_7_and_up",
+                         "LCR-hs38",
+                         "mappability=0",
+                         "exome"]
+
+        Array[String] jexlVariantSelectors = ["vc.isSimpleIndel()  && vc.getIndelLengths().0<0", "vc.isSimpleIndel() && vc.getIndelLengths().0>0"]
+        Array[String] variantSelectorLabels = ["deletion","insertion"]
+    }
+
+    Int VCF_disk_size = ceil(size(input_callset, "GiB")) + 10
+    File monitoring_script="gs://broad-dsde-methods-monitoring/cromwell_monitoring_script.sh"
```

Comment:
this means that if you don't have access to this bucket you can't run the workflow, right?  I think better to make it an input to the workflow, so someone w/o access can still use.  And adding a comment that this script can be found in the `Scripts/monitoring` directory of this repo

Reply:
Done